blog/__init__.py

```diff
@@ -0,0 +1 @@
+"""Initializes the app."""
```

Comment:
Don't omit parts of phrases, like "This package" or "This module". Use full sentences.

Reply:
Marking as resolved.